src/object.js

```diff
@@ -127,6 +127,15 @@ define([], function() {
 		return allSubclasses;
 	};
 
+	/**
+	 * Return all concrete subclasses.
+	 */
+	object.allConcreteSubclasses = function() {
+		return this.allSubclasses().filter(function(klass) {
+			return !klass.isAbstract;
```

Comment:
I would do a `forEach` again, to avoid iterating twice.

The code might be a bit duplicated, but it's quite a performance improvement (`Object` will have quite a shitload of subclasses)